src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -1084,13 +1059,6 @@ async def _get_tx_seg_genomic_metadata(
             tx_ac = mane_data["RefSeq_nuc"]
             grch38_ac = mane_data["GRCh38_chr"]
 
-        # Always liftover to GRCh38
-        genomic_ac, genomic_pos, err_msg = await self._get_grch38_ac_pos(
-            genomic_ac, genomic_pos, grch38_ac=grch38_ac
-        )
-        if err_msg:
-            return GenomicTxSeg(errors=[err_msg])
```

Comment:
So I'm kind of confused on the purpose of this helper method. It seems like some of the work already exists in `_genomic_to_tx_segment`?

Reply:
I agree. I'm going to look at this method to see if I can remove any duplicate code